src/components/TopBar/TopBar.tsx

```diff
@@ -107,17 +107,12 @@ export const TopBar: React.FunctionComponent<TopBarProps> & {
     );
   }
 
-  const searchResultsMarkup =
-    searchResults && searchResultsVisible ? (
-      <Search visible={searchResultsVisible} onDismiss={onSearchResultsDismiss}>
```

Comment:
There was once code to ensure this component still rendered even when it wasn't _visible_.
This was because there result services loading and queries fired when the QuickSearch component mounted.
At some point this changed to render null when `searchResultsVisible` was false, forcing a remount, refetch, requery each time the search is re-opened.

At some point in the near future we are reworking the QuickSearch component so it no longer relies on mount lifecycle methods, but for now, this would be a slight perf improvement and help mitigate some bugs.